plugins/webhooks/github.js

```diff
@@ -173,8 +175,8 @@ function pullRequestEvent(request, reply) {
     const prNumber = hoek.reach(payload, 'pull_request.number');
     const repository = hoek.reach(payload, 'pull_request.base.repo.ssh_url');
     const branch = hoek.reach(payload, 'pull_request.base.ref');
-    const scmUrl = `${repository}#${branch}`;
-    const prRef = `${repository}#pull/${prNumber}/merge`;
+    const scmUrl = scmGithub.formatScmUrl(`${repository}#${branch}`);
```

Comment:
It's annoying how I had to instantiate an object specifically for this function, which is another reason why I was adverse to changing to this way. Not that it isn't correct, but using it is awkward to implement for a case like this.